cortex-js/README.md

```diff
@@ -11,91 +11,49 @@
 > ⚠️ **Cortex is currently in Development**: Expect breaking changes and bugs!
 
 ## About
-Cortex is an openAI-compatible local AI server that developers can use to build LLM apps. It is packaged with a Docker-inspired command-line interface and a Typescript client library. It can be used as a standalone server, or imported as a library. 
+Cortex is an OpenAI-compatible AI engine that developers can use to build LLM apps. It is packaged with a Docker-inspired command-line interface and client libraries. It can be used as a standalone server or imported as a library. 
 
-Cortex currently supports two inference engines:
+Cortex currently supports 3 inference engines:
 
 - Llama.cpp
+- ONNX Runtime
 - TensorRT-LLM
 
-> Read more about Cortex at https://jan.ai/cortex
-
 ## Quicklinks
-Cortex
-- [Website](https://jan.ai/)
-- [GitHub](https://github.com/janhq/cortex)
-- [User Guides](https://jan.ai/cortex)
-- [API reference](https://jan.ai/api-reference)
-
-## Prerequisites
-
-### **Dependencies**
-
-Before installation, ensure that you have installed the following:
-- **Node.js**: version 18 and above is required to run the installation.
-- **NPM**: Needed to manage packages.
-- **CPU Instruction Sets**: Available for download from the [Cortex GitHub Releases](https://github.com/janhq/cortex/releases) page.
-
-
->💡 The **CPU instruction sets** are not required for the initial installation of Cortex. This dependency will be automatically installed during the Cortex initialization if they are not already on your system.
-
 
-### **Hardware**
+- [Homepage](https://cortex.jan.ai/)
+- [Docs](https://cortex.jan.ai/docs/)
 
+## Quickstart
+### Prerequisites
 Ensure that your system meets the following requirements to run Cortex:
-
+- **Dependencies**:
+  - **Node.js**: version 18 and above is required to run the installation.
+  - **NPM**: Needed to manage packages.
+  - **CPU Instruction Sets**: Available for download from the [Cortex GitHub Releases](https://github.com/janhq/cortex/releases) page.
 - **OS**:
   - MacOSX 13.6 or higher.
   - Windows 10 or higher.
-  - Ubuntu 22.04 and later.
-- **RAM (CPU Mode):**
-  - 8GB for running up to 3B models.
-  - 16GB for running up to 7B models.
-  - 32GB for running up to 13B models.
-- **VRAM (GPU Mode):**
+  - Ubuntu 12.04 and later.
```

Comment:
22.04 and later

Reply:
Required to run Tensorrt-LLM engine (GLIB compatibility)

Reply:
What do you mean by Required to run Tensorrt-LLM engine (GLIB compatibility)? Is it required for every user to run Tensorrt-LLM engine or only for ubuntu?

Reply:
I checked with guys, 20.04 for llama.cpp (default), and 22.04 is recommended for every user to run Tensorrt-LLM engine on ubuntu

Reply:
I mean better leave 22.04 for now (Ubuntu)

Reply:
I see. Okay, I have updated the README per your feedback. Kindly check it again